dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractCommandExecutor.java

```diff
@@ -327,7 +327,7 @@ private void collectPodLogIfNeeded() {
             ThreadUtils.sleep(SLEEP_TIME_MILLIS * 5L);
             try (
                     LogWatch watcher = ProcessUtils.getPodLogWatcher(taskRequest.getK8sTaskExecutionContext(),
-                            taskRequest.getTaskAppId())) {
+                            taskRequest.getTaskAppId(), "")) {
```

Comment:
Why is it an empty string?

Reply:
In `AbstractCommandExecutor`, only need to get log watcher for Spark on K8S task, and we directly collect logs from driver pod, however, we can not set container name during task launching. Thus, we set empty string and the effect is same as collecting logs from all containers.